examples/vanilla-ts-esm/public/mux-uploader-widget.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Basic &lt;mux-uploader/&gt; example</title>
+    <script type="module" src="./dist/mux-uploader.js"></script>
+    <style>
+      body {
+        margin: 0;
+        background: #e8e8e8;
+      }
+
+      .direct-upload-url {
+        padding: 8px 12px;
+        width: 400px;
+      }
+
+      .container {
+        box-sizing: border-box;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        font-family: Arial;
+        height: 90vh;
+      }
+
+      .dashboard {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-evenly;
+        flex-wrap: wrap;
+      }
+
+      .widget {
+        margin-bottom: 30px;
+        padding: 20px;
+        height: 250px;
+        width: 500px;
+        border-radius: 20px;
+        background-color: #f9f9f9;
+        box-sizing: border-box;
+        box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
+
+        display: flex;
+        justify-content: center;
+        align-items: center;
+
+        position: relative;
+      }
+
+      header {
+        justify-content: center;
+        text-align: center;
+      }
+
+      header > h1 {
+        font-size: 5vw;
+        line-height: 6vw;
+        text-align: left;
+        max-width: 45vw;
+      }
+
+      h2 {
+        font-size: 3vw;
+        line-height: 4vw;
+        text-align: left;
+        max-width: 45vw;
+      }
+
+      input {
+        max-width: 45vw;
+        margin-bottom: 50px;
+      }
+
+      footer {
+        box-sizing: border-box;
+        width: 100%;
+        padding: 20px;
+      }
+      
+      mux-uploader {
+        --button-border-radius: 40px;
+      }
+     
+      mux-uploader[upload-in-progress] {
+        width: inherit;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <h2>Enter your upload GCS url:</h1>
+        <input type="text" class="direct-upload-url" placeholder="https://storage.googleapis.com/..." />
+      </header>
+      <div class="dashboard">
+        <div class="widget"></div>
+        <div class="widget">
+          <mux-uploader type="bar" id="uploader-controller" disableDrop="true">
```

Comment:
```suggestion
          <mux-uploader type="bar" id="uploader-controller" disable-drop>
```

an HTML attribute without a value implies a boolean true

Reply:
yeah that's what I thought too so I was surprised when the logic wouldn't recognize the attribute value as truthy without explicit `true` 🤔  maybe I imagined it

Reply:
for boolean attributes, you should be able to rely on `myFancyCustomElement.hasAttribute('my-special-boolean')`